packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -1,7 +1,96 @@
-library vector_graphics;
+import 'dart:typed_data';
+import 'dart:ui' as ui;
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+import 'package:vector_graphics_codec/vector_graphics_codec.dart';
 
-/// A Calculator.
-class Calculator {
-  /// Returns [value] plus 1.
-  int addOne(int value) => value + 1;
+import 'src/listener.dart';
+
+const VectorGraphicsCodec _codec = VectorGraphicsCodec();
+
+/// A widget that displays a vector_graphics formatted asset.
+class VectorGraphicsWidget extends StatefulWidget {
```

Comment:
Maybe just `VectorGraphic`?

Reply:
Also, I have a slight preference to just making this a `LeafRenderObjectWidget` and setting the picture on the render object when it's available which triggers a markNeedsPaint.

This will also want to handle some sizing and fitting options, as well as color setting (but maybe not color filtering).